packages/panzer/adapters-stk/example/main_driver/energy-ss-tp-multiblock-ic-bc-issue.xml

```diff
@@ -0,0 +1,310 @@
+<ParameterList name="Drekar">
+
+    <ParameterList name="Mesh">
+       
+        <Parameter name="Source" type="string" value="Exodus File" />
+
+        <ParameterList name="Exodus File">
+            <Parameter name="File Name" type="string" value="tri3-combined-centers.exo" />
+        </ParameterList>
+
+    </ParameterList>
+
+    <ParameterList name="Initial Conditions">
+        <ParameterList name="left">
+            <ParameterList name="TEMPERATURE">
+                <Parameter name="Value" type="double" value="0.0"/>
+            </ParameterList>
+        </ParameterList>
+        <ParameterList name="right">
+            <ParameterList name="TEMPERATURE">
+                <Parameter name="Value" type="double" value="1.0"/>
+            </ParameterList>
+        </ParameterList>
+    </ParameterList>
```

Comment:
@rppawlo So far I've just decomposed the mesh as they instructed and run with inconsistent IC's on the two blocks. That does not seem to give a wonky solution, see attached. 
![Screenshot 2025-02-21 at 4 16 56 PM](https://github.com/user-attachments/assets/6baec3c3-a1fe-4139-9526-d791e2e2ceb9)

Reply:
Did you rerun with 2 or 3 mpi processes? I think 1 looks correct. You may have to check that the initial conditions are printed at the right time. The unit test required some work on IO to be able to reproduce the issue.

Reply:
@rppawlo This is on 3 mpi processes. I didn't do any of the IO work to visualize the IC; however, I thought we determined that the BC issue should persist to the steady-state solution. I'm not seeing it at the moment.

Reply:
<img width="1767" alt="Screenshot 2025-02-26 at 1 13 52 PM" src="https://github.com/user-attachments/assets/5bb51e8e-7357-40e6-9198-f8659f854656" />

@rppawlo This is on 3 MPI processes with interval 7. This is the converged solution. I don't fully understand what is different between this and the case I ran the other day so need to do some digging.